news/2023/2023-02-09-skin-of-the-year-2022.md

```diff
@@ -4,23 +4,25 @@ title: "Skin of the Year 2022: Voting Open"
+Ever since skinning was first introduced to osu! many years ago, a sprawling and ever so creative community has been evolving and expanding around it. Many thousands of skins have been created in the past 15 years with a myriad of styles and artists coming and going, but until now the community has never had an official vote to determine the best made in a given year.
 
-## Voting details
+Now's your chance to become a part of history in this first-of-a-kind skin competition!
 
-All skins that have been released during 2022 and added to [the compendium](https://compendium.skinship.xyz/) are eligible to compete, **including contest & podium entries**. The top-voted entry will receive a unique profile badge based on their entry and 3 months of osu!supporter. Entries #2 – #10 will receive 1 month of osu!supporter.
+## Voting details
 
-You might ask yourself "Why would I bother voting when [YUGEN](https://osu.ppy.sh/community/forums/topics/365036?n=1) is the best?" — while that skin is really cool, we urge you to check out the sheer amount of creativity that various artists have put into their works throughout the year with **290** skins that have been submitted to the [skinning forums](https://osu.ppy.sh/community/forums/15) and **136** of them being added to the compendium, all of them waiting to grab your attention, and maybe even become your main skin.
+All skins that have been released during 2022 and were added to [the compendium](https://compendium.skinship.xyz/) are eligible to compete, **including contest & podium entries**. Everyone gets **15** votes each to cast, and you can spend as many or as little as you like.
 
-If these facts managed to intrigue you enough to the point of wanting to peruse around the contest page and select the skins you like the most:  
+The top-voted entry will receive a unique profile badge based on their entry and 3 months of osu!supporter for their victory, while the remaining finalists will receive 1 month of osu!supporter each.
 
-[**Click here to vote for the Skin of the Year!**](https://osu.ppy.sh/community/contests/170)
+We urge you to check out the sheer amount of creativity that various artists have put into their works throughout the year, with **290** skins that have been submitted to the [skinning forums](https://osu.ppy.sh/community/forums/15) (and **136** of them being added to the compendium) all just waiting to make their home in your client as your new main skin.
+  
+[**Click here to get involved and vote for 2022's Skin of the Year!**](https://osu.ppy.sh/community/contests/170)
 
-You will be able to cast your vote for **14 days** after this post goes up, with the votes closing on **23rd of February**. Hopefully that's enough time to go through the catalogue and pick your favourite.
+The votes will remain open for **14 days** after this post goes up, closing officially on the **23rd of February**. Hopefully, that should be enough time to go through the catalogue and pick your favourites.
```

Comment:
shouldnt this say voting instead of votes

Reply:
it should, and now does